<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gamemap</class>
 <widget class="QWidget" name="Gamemap">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GAMEMAP</string>
  </property>
  <widget class="QPushButton" name="Battle1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(146, 208, 80);
}</string>
   </property>
   <property name="text">
    <string>史莱姆</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelmap1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>851</width>
     <height>771</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Pictures/map1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleSkeleton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>592</y>
     <width>61</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(146, 208, 80);
}</string>
   </property>
   <property name="text">
    <string>骷髅</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleEyeofCthulu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(112, 48, 160);
}</string>
   </property>
   <property name="text">
    <string>克苏鲁之眼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Battle_11451481019">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>270</y>
     <width>74</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(112, 48, 160);
}</string>
   </property>
   <property name="text">
    <string>野兽
先辈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleMachine">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>216</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(0, 176, 240);
}</string>
   </property>
   <property name="text">
    <string>机皇帝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleMushroom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(0, 176, 240);
}</string>
   </property>
   <property name="text">
    <string>蘑菇怪</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleArthas">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>393</y>
     <width>126</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
border-radius:12px;
background-color: rgb(226, 107, 10);
};
</string>
   </property>
   <property name="text">
    <string>阿尔萨斯</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Old English Text MT</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MAP A&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Stencil</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 127);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ERANGEL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextmap">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>NEXT MAP</string>
   </property>
  </widget>
  <widget class="QLabel" name="EDagger">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>576</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>影之匕首：所有打出的威力大于等于24的单次攻击威力增加8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 107, 10);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egainlife">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>血脉项链：每场战斗结束后MAXHP永久加2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 48, 160);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egainattack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>423</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>增幅宝珠：ATK永久加5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 176, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egaincard">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>321</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>法师的睿智：抽牌数永久加1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 48, 160);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egaindefense">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>474</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>圣物之盾：DEF永久加2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 176, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Egaincost">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>372</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>魔力宝石：每回合获得MANA永久加2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 48, 160);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ERhoAias">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>525</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Rho Aias：所有受到的伤害大于等于20的单次攻击威力减少5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 107, 10);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>labelmap1</zorder>
  <zorder>Battle1</zorder>
  <zorder>BattleSkeleton</zorder>
  <zorder>BattleEyeofCthulu</zorder>
  <zorder>Battle_11451481019</zorder>
  <zorder>BattleMachine</zorder>
  <zorder>BattleMushroom</zorder>
  <zorder>BattleArthas</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>nextmap</zorder>
  <zorder>EDagger</zorder>
  <zorder>Egainlife</zorder>
  <zorder>Egainattack</zorder>
  <zorder>Egaincard</zorder>
  <zorder>Egaindefense</zorder>
  <zorder>Egaincost</zorder>
  <zorder>ERhoAias</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
